extensions/resource-deployment/notebooks/bdc/2019/CTP3-1/deploy-bdc-existing-cluster.ipynb

```diff
@@ -61,7 +61,7 @@
         },
         {
             "cell_type": "code",
-            "source": "import getpass\ndef get_user_input(input_name, is_password = False):\n    if is_password:\n        user_input = getpass.getpass(prompt = input_name)\n        print(input_name + ': ***********')\n    else:\n        user_input = input(input_name)\n        print(input_name + ': ' + user_input)\n    if user_input == \"\":\n        raise SystemExit(f'{input_name} is required')\n    return user_input;\n\nmssql_cluster_name = get_user_input('Cluster name')\nbdc_controller_username = get_user_input('Controller username')\nbdc_controller_password = get_user_input('Controller password', True)\nbdc_mssql_sa_password = get_user_input('SQL Server password', True)\ndocker_username = get_user_input('Docker username')\ndocker_password = get_user_input('Docker password', True)",
+            "source": "import getpass\ndef get_user_input(input_name, is_password = False, confirm_password = False):\n    if is_password:\n        user_input = getpass.getpass(prompt = input_name)\n        if confirm_password:\n            user_input_confirm = getpass.getpass(prompt = 'Confirm '+ input_name)\n            if user_input != user_input_confirm:\n                raise SystemExit(f'{input_name} does not match the confirmation password')\n        print(f'{input_name}: ******')\n    else:\n        user_input = input(input_name)\n        print(input_name + ': ' + user_input)\n    if user_input == \"\":\n        raise SystemExit(f'{input_name} is required')\n    return user_input;\n\nmssql_cluster_name = get_user_input('Cluster name')\nbdc_controller_username = get_user_input('Controller username')\nbdc_controller_password = get_user_input('Controller password', True, True)\nbdc_mssql_sa_password = get_user_input('SQL Server password', True, True)\ndocker_username = get_user_input('Docker username')\ndocker_password = get_user_input('Docker password', True)",
```

Comment:
Currently we only support single password for both controller and mssql_sa. Can you update that too.